apps/masterbots.ai/app/api/wordware/describe/route.ts

```
@@ -1,14 +1,15 @@
+import { WordWareDescribeDAtaResponse } from '@/types/wordware-flows.types'
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)